crits/standards/parsers.py

```diff
@@ -160,45 +253,179 @@ def parse_observables(self, observables):
+                    self.parse_res(imp_type, obs, res)
+                elif isinstance(item, File) and self.has_network_artifact(item):
+                    imp_type = "PCAP"
+                    description = str(item.description)
+                    filename = str(item.file_name)
+                    data = None
+                    for obj in item.parent.related_objects:
+                        if (isinstance(obj.properties, Artifact) and
+                            obj.properties.type_ == Artifact.TYPE_NETWORK):
+                            data = base64.b64decode(obj.properties.data)
+                    res = handle_pcap_file(filename,
+                                           data,
+                                           self.source,
+                                           user=analyst,
+                                           description=description)
+                    self.parse_res(imp_type, obs, res)
+                elif isinstance(item, File):
+                    imp_type = "Sample"
+                    filename = str(item.file_name)
+                    md5 = item.md5
+                    data = None
+                    for obj in item.parent.related_objects:
+                        if (isinstance(obj.properties, Artifact) and
+                            obj.properties.type_ == Artifact.TYPE_FILE):
+                            data = base64.b64decode(obj.properties.data)
+                    res = handle_file(filename,
+                                      data,
+                                      self.source,
+                                      user=analyst,
+                                      md5_digest=md5,
+                                      is_return_only_md5=False)
+                    self.parse_res(imp_type, obs, res)
+                elif isinstance(item, EmailMessage):
+                    imp_type = "Email"
+                    data = {}
+                    data['source'] = self.source.name
+                    data['source_method'] = self.source_instance.method
+                    data['source_reference'] = self.source_instance.reference
+                    data['raw_body'] = str(item.raw_body)
+                    data['raw_header'] = str(item.raw_header)
+                    data['helo'] = str(item.email_server)
+                    if item.header:
+                        data['message_id'] = str(item.header.message_id)
+                        data['subject'] = str(item.header.subject)
+                        data['sender'] = str(item.header.sender)
+                        data['reply_to'] = str(item.header.reply_to)
+                        data['x_originating_ip'] = str(item.header.x_originating_ip)
+                        data['x_mailer'] = str(item.header.x_mailer)
+                        data['boundary'] = str(item.header.boundary)
+                        data['from_address'] = str(item.header.from_)
+                        data['date'] = item.header.date
```

Comment:
This line is leading to error: 
`Observable: Failed to save object. <br /><pre>'DateTime' object has no attribute 'read'</pre>`

`item.header.date` is a `cybox.common.properties.DateTime object`. Because it isn't a standard datetime.datetime, the CRITs Email class "_custom_save" method tries to use dateutil.parser, but it chokes.
Changing `data['date'] = item.header.date` to `data['date'] = item.header.date.value` seems to help, but again, I'm not certain that is the best solution.